### PR TITLE
Feature/create userid endpoint

### LINKED ISSUE #5: Create /users/{id} endpoont
- [x] Crear un endpoint que reponda con la información del usuario indicado en los `params`
- [x] El usuario debe ser cargados por medio del metodo `findOne()`
- [x] Si el usuario no existe responder con {message: "Usuario no encontrado"}
- [x] Hacer que por medio de un parametro en el header de la peticion los mensajes de error se respondan con diferentes idiomas

### PR DESCRIPTION
**Pull request by ceroncmcr**
The requested changes were made for the issue
Close #5

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -5,3 +5,4 @@ DB_HOST=
 DB_NAME=
 DB_PORT=
 NEW_ENV=
```
Comment (oicrruf) [inline .env.example]: Esta línea se puede eliminar del archivo .env original y al sincronizar en el commit verifica si se actualiza

**Review by MarkRiv — APPROVED** [review]

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+    ports:
+      - 5432:5432
+    volumes:
+      - ./data:/var/lib/postgresql/data
+
   pgadmin:
     image: dpage/pgadmin4
     environment:
       - PGADMIN_DEFAULT_EMAIL=mrivas@mail.com
```
Comment (oicrruf) [inline docker-compose.yaml]: Coordinate with @MarkRiv to remove these data from this file

- [x] PGADMIN_DEFAULT_EMAIL
- [x] PGADMIN_DEFAULT_PASSWORD

File: routes/user.router.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+router.get(
+  '/:id',
+  validatorHandler(getUserSchema, 'params'),
+  async (req, res) => {
+    try {
+      const { id } = req.params;
+      const user = await service.findByPk(id);
+      if (!user) {
+        res.status(404).json({
+          message: 'Usuario no encontrado',
```
Comment (oicrruf) [inline routes/user.router.js]: This is the type of message we should put in the language JSON, the validation and JOI generated messages that are translated by the package

File: schemas/errorMessage.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "en": {
+    "any.required": "unknown message in English",
+    "number.base": "\"id\" must be a number"
+  },
+  "es": {
+    "any.required": "unknown message in English",
+    "number.base": "\"id\" debe ser un numero"
+  }
```
Comment (oicrruf) [inline schemas/errorMessage.json]: Add messages in English or Spanish for "User not found" and "Usuario no encontrado".

**Review by oicrruf — CHANGES_REQUESTED** [review]

**Review by oicrruf — APPROVED** [review]

**Pull request merged**